be2-scala/src/test/scala/util/examples/Witness/WitnessMessageExamples.scala

```diff
@@ -0,0 +1,53 @@
+package util.examples.Witness
+
+import ch.epfl.pop.json.MessageDataProtocol._
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.model.network.method.message.data.witness.WitnessMessage
+import ch.epfl.pop.model.objects._
+import com.google.crypto.tink.subtle.Ed25519Sign
+import spray.json._
+
+
+object WitnessMessageExamples {
+
+  val keyPair: Ed25519Sign.KeyPair = Ed25519Sign.KeyPair.newKeyPair
```

Comment:
That's the only test that generates a dynamic keypair. It would be nicer to use a well-known one:

```diff
--- a/be2-scala/src/test/scala/util/examples/Witness/WitnessMessageExamples.scala
+++ b/be2-scala/src/test/scala/util/examples/Witness/WitnessMessageExamples.scala
@@ -4,15 +4,15 @@ import ch.epfl.pop.json.MessageDataProtocol._
 import ch.epfl.pop.model.network.method.message.Message
 import ch.epfl.pop.model.network.method.message.data.witness.WitnessMessage
 import ch.epfl.pop.model.objects._
-import com.google.crypto.tink.subtle.Ed25519Sign
+import util.examples.data.{TestKeyPairs, KeyPairWithHash}
 import spray.json._


 object WitnessMessageExamples {

-  val keyPair: Ed25519Sign.KeyPair = Ed25519Sign.KeyPair.newKeyPair
-  val privateKey: PrivateKey = PrivateKey(Base64Data.encode(keyPair.getPrivateKey))
-  val SENDER: PublicKey = PublicKey(Base64Data.encode(keyPair.getPublicKey))
+  val keyPair: KeyPairWithHash = TestKeyPairs.keypairs(1)
+  val privateKey: PrivateKey = keyPair.keyPair.privateKey
+  val SENDER: PublicKey = keyPair.keyPair.publicKey

Reply:
Probably a good idea for consistency 👍